spec/yaml/swiftnav/sbp/skylark.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.skylark
+description: Messages from the Skylark corrections service.
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+  - MSG_ACKNOWLEDGE:
+      id: 0x0FA0
+      short_desc: Acknowledgement message in response to a request for corrections
+      fields:
+        - request_counter:
+            type: u8
+            desc: Echo of the request counter field from the corresponding CRA message, or 255 if no request counter was provided.
```

Comment:
To me a "counter" implies a sequence that starts at zero and I'm not sure that that's important for this message, I've proposed to System Design that this be reword/recast as a "request ID" so that we don't have the implied behavior of a counter-- do you have thoughts on this?

Reply:
I agree, especially considering we just added `stream_counter` and `on_demand_counter` which behavior exactly as you describe, I'll see about changing the documentation